assets/javascripts/discourse/templates/components/chat-emoji-avatar.hbs

```diff
@@ -0,0 +1,5 @@
+<div class="chat-emoji-avatar">
```

Comment:
What's a "chat-emoji-avatar"? I mean, the words make sense separately to me but is it an avatar that's actually an emoji? Being a little pedantic here, since we're renaming things

Reply:
Self-answered below